guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,196 @@
 {
```

Comment:
<blockquote>Enhances&nbsp;<code><a href="https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/files/" target="_blank">SingleShotDetector</a></code>&nbsp;to support rotated bounding boxes as inputs</blockquote><p>This support has not been added yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/583/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Removed</p>